src/crimson/os/alienstore/alien_store.h

```diff
@@ -66,6 +66,14 @@ class AlienStore final : public FuturizedStore,
     const ghobject_t &oid,     ///< [in] oid
     const std::optional<std::string> &start ///< [in] start, empty for begin
     ) final; ///< @return <done, values> values.empty() iff done
+  
+  read_errorator::future<std::tuple<bool, omap_values_t>> log_get_values(
```

Comment:
Why do alienstore and cyanstore need to implement `log_get_values()` ? Isn't it already guarded by the false `has_log_store()` ?

If it isn't expected to be used, guard by ceph_abort().

Reply:
Done.